torch/_torch_docs.py

```diff
@@ -970,6 +970,64 @@ def merge_dicts(*dicts):
 Alias for :func:`torch.atanh`.
 """)
 
+add_docstr(torch.asarray,
+           r"""
+asarray(obj, *, dtype=None, device=None, copy=None, requires_grad=False) -> Tensor
+
+Creates a new :class:`Tensor` out of :attr:`obj` storage, by either copying or
+aliasing it.
+
+:attr:`obj` will be tested for, in order:
+
+1. a :class:`Tensor` object (copy and alias allowed)
+2. a DLPack object (copy and alias allowed)
+3. a Python object that implements the buffer protocol (copy and alias allowed)
+4. an arbitrary sequence (only copy allowed)
+
+When :attr:`copy` is not given, this function will try first to alias the given
```

Comment:
same as above

Reply:
I agree with @anjali411 that "view" is more familiar than the term "alias" here. But I also think this detail can be moved into the arg's description below